python_modules/dagster/dagster/_core/definitions/decorators/definitions_decorator.py

```diff
@@ -0,0 +1,19 @@
+from dagster._core.definitions.definitions_loader import DefinitionsLoader, DefinitionsLoadFn
+
+
+def definitions(fn: DefinitionsLoadFn) -> DefinitionsLoader:
```

Comment:
So let's consider what this actually is.

It is a decorator to mark a function as discoverable and loadable by our tools. If we have broader ambitions for this decorator we should say so. I don't think this docblock or name accurately describes that. Perhaps consider these alternatives (not even saying these are good ideas.

`@entrypoint`
`@loadable`
`@defs`
`@codelocation`

This docblock should also describe behavior if there is more than one `@definitions`-marked function in the module.

```python
"""
@definitions explicitly marks a function as loadable by Dagster tools. Tools such as dagster-webserver the dagster cli when loading a module search for a function decorated with @definitions and invoke it in order to produce a Definitions object.
"""
```

And so on and so forth.

Reply:
Btw I've convinced myself that `@definitions` is a reasonable name writing that out, but we should take a first pass at a legit docblock.